### DISCUSSION TITLE
How to automatically load the track when I open the jbrowse

### DISCUSSION THREAD

**Discussion by zhangyw0713** [GMOD/jbrowse-components#1939] [Q&A]
We previously constructed a genome browser based on Jbrowse, but we cannot find the parameter that can be used to automatically load the track when the browser was opened. For example, we configured 4 tracks (track 1-4) for the browser. In default, the tracks were not loaded if we do not check them. 
If we want to show the users track 1 and 2 (i.e. track 1 and 2 should be automatically checked) when they open the browser. What parameter should we set? 
That's my first time constructing JBrowse. Really hope to get some suggestions and help from all you experts. 
Thanks very much!

**Comment by garrettjstevens** [comment]
Hi @zhangyw0713,

If you want to have JBrowse default to a certain state (e.g. Linear Genome View with two tracks open), you can use the "default session." This is an entry in your config that tells JBrowse what state to be in when it opens.

I'm not sure how you set up your config file, but if you're using the admin-server, the instructions for setting a default session are here: https://jbrowse.org/jb2/docs/quickstart_gui#setting-a-default-session.

If you're using the JBrowse CLI, you can use the `set-default-session` command, which is documented here: https://github.com/GMOD/jbrowse-components/tree/main/products/jbrowse-cli#jbrowse-set-default-session. You can generate the session.json used in this command by getting JBrowse into the state you want and then selecting `File -> Export session`.

Let us know if you have any more questions.
